app/models/subheading.rb

```diff
@@ -21,6 +21,16 @@ class Subheading < GoodsNomenclature
                 :formatted_description,
                 :declarable
 
+  def descendants
+    included_subheading = commodities.find { |c| c.goods_nomenclature_sid == goods_nomenclature_sid }
+
+    if included_subheading.present?
```

Comment:
this could just be a ternary? or even safe navigation, eg

```
included_subheading ? included_subheading.children : []
```

```
included_subheading&.children || []
```